python/moose/SBML/readSBML.py

```diff
@@ -1,20 +1,34 @@
-# -*- coding: utf-8 -*-
```

Comment:
Keep this line whenever possible in python scripts. https://www.python.org/dev/peps/pep-0263/